### ISSUE TITLE
IfcOrientationExpression needs better documentation

### ISSUE THREAD

**Issue by I-Sokolov** [buildingSMART/IFC_UML#63]
The description does not clarify what are these directions
https://standards.buildingsmart.org/IFC/DEV/IFC4_3/RC1/HTML/link/ifcorientationexpression.htm

**Comment by TLiebich** [comment]
yes, this is a known deficiency. It was addressed during the Infrastructure Extension Project.
@SergejMuhic, @pjanck - do you remember the outcome and why it misses the inclusion into the documentation?

@I-Sokolov - it's not an IFC5 issue, we need a solution earlier. 
currently those issues leading to an improved schema and/or documentation of IFC4.3 are collected here:
https://forums.buildingsmart.org/c/rooms/infra-commonschematech/57

**Comment by TLiebich** [comment]
or better here (?):
https://github.com/buildingSMART/IFC4.3.x-development

**Comment by SergejMuhic** [comment]
I have created the proposal for a documentation update at the end of Common Schema and we wanted to finalize it in Infra Deployment. Bentley is currently not participating but would be very welcome. @I-Sokolov if I cannot convince you to participate :), your input would be very welcome. In that case, I would send you the word document. Please let me know.

In the meantime, the directions are specified in the local coordinate system of the IfcLinearPositioningElement of which the Axis is being used for measurements. So, both IfcDirection attributes would be relative to the IfcLinearPositioningElement.ObjectPlacement coordinate system.

**Comment by SergejMuhic** [comment]
Oh man, just realized the problem by looking at the existing documentation. The directions should be specifying y and z!

**Comment by I-Sokolov** [comment]
@SergejMuhic, I am trying to connect with your by bSI forums, GitHub does not allow direct messages
I would glad to attend Infra Deployment, even I am dummy in the area (we started 4x import some weeks ago) but other Bentley mates have infrastructure experience.

But now I am trying to understand IfcOrientationExpression.
Which values are you expecting for this example
![image](https://user-images.githubusercontent.com/6941925/89211855-1e33a580-d5cb-11ea-89c5-0f416c43f5f4.png)

**Comment by pjanck** [comment]
Following the publication of `IFC4X3_RC2` this issue can be retired. See the newly proposed `IfcPointByDistanceExpression` [here](https://standards.buildingsmart.org/IFC/DEV/IFC4_3/RC2/HTML/link/ifcpointbydistanceexpression.htm).
